source/_integrations/pvpc_hourly_pricing.markdown

```diff
@@ -18,59 +18,33 @@ This sensor uses the official API to get the hourly price of electricity in Spai
-with a contracted power not exceeding 10 kW and which are under the PVPC 
+with a contracted power not exceeding 15 kW and which are under the PVPC 
 (Voluntary Price for Small Consumer).
 
 It includes the energy term of the access tolls, the charges and the production cost. It does not include taxes.
-The hourly prices are the same throughout the Spanish territory regardless of the time zone.
-
-It can be set up via the integrations panel in the configuration screen.
+The hourly prices and energy periods are the same throughout the Spanish territory regardless of the time zone, 
+except for the cities of Ceuta and Melilla, where they are slightly different.
 
 <iframe src="https://www.esios.ree.es/en/embed/active-energy-invoicing-price-pvpc" width="100%" height="608"></iframe>
 
 More information available at http://www.cnmc.es/en/ and http://www.omie.es/en/
 
 ## Configuration
 
-To configure PVPC Hourly Pricing, you can set it up via the integrations panel in the configuration screen.
-
-Set a name for the price sensor (default is `sensor.pvpc`) and select one of the three tariffs, 
-corresponding to your contracted rate, according to the number of billing periods per day 
-(one / peak + valley / shifted peak + valley).
-
-- 1 period: `normal`, for the "Default PVPC tariff, (2.0 A)".
-- 2 periods: `discrimination`, for the "Efficiency 2 periods (2.0 DHA)", with 10h peak interval from 11:00 UTC to 21:00 UTC.
-- 3 periods: `electric_car`, for the "Electric vehicle tariff (2.0 DHS)", optimized for electric car owners to charge at night.
-
-The default is `discrimination`, with 2 periods, as it is usually the cheapest one for home consumers. 
-
-You can add multiple sensors (up to 3, one per tariff) by adding them again through the integrations panel.
-
-### Advanced configuration
+To configure PVPC Hourly Pricing, set it up via the integrations panel in the configuration screen.
 
-PVPC Hourly Pricing allows manual configuration by adding a section to your `configuration.yaml`. 
+Set a name for the price sensor (default is `sensor.pvpc`), and select one of the two available tariffs, 
+according to your geographic position in Spain: 
+- `2.0TD`, for the Peninsula, the Balearic Islands and the Canary Islands.
+- `2.0TD (Ceuta/Melilla)`, for the cities of Ceuta and Melilla.
 
-```yaml
-# Set up electricity price sensors as a component:
-pvpc_hourly_pricing:
-  - name: PVPC manual ve
-    tariff: electric_car
-  - name: PVPC manual nocturna
-    tariff: discrimination
-```
+Set also your contracted power (in kW) for the two power periods that apply with the new 2.0TD tariff
+(one for P1/P2 and the other one for the valley period, P3), to show your available electric power as a sensor attribute.
 
-{% configuration %}
-name:
-  description: Custom name for the sensor.
-  required: true
-  type: string
-tariff:
-  description: Contracted electric tariff.
-  required: true
-  default: discrimination
-  type: string
-{% endconfiguration %}
```

Comment:
This change is not related to the parent PR and thus does not belong in this PR.

Should this be changed in our current documentation?

Reply:
Hi @frenck, you're totally right. I doubted with the PR destination (next or current), and how to change both in the same time, in sync with the parent PR.

The thing is the parent PR _could_ remove the `CONFIG_SCHEMA` and `async_setup`, as **they are already deprecated in 2021.6.0** (I think for 2021.7 should be removed, right?). Because, if not, then the change in the PR should be also reflected in the `CONFIG_SCHEMA` (the config_entry.data is changed), but I thought it was an overkill for something already deprecated, don't you agree?

But I have no problem in changing anything, splitting this one in 2, add another commit removing the yaml config in parent, open another in core for the yaml config removal... you name it ;-)

Reply:
We generally deprecated the config schema for at least 2 release cycles.
Nevertheless, this is not related to the parent PR, so please move it out of the scope.

Reply:
> We generally deprecate the config schema for at least 2 release cycles.
Ah, I thought it was just 1, sorry!

Then, instead of moving out the removal, what I need to do _here_ is to adapt it to the new schema that the parent PR introduces (so, editing it, not removing), and then would be inside the scope again, right?

Reply:
@frenck, I just restored the YAML section, updating the fields to sync with the new schema introduced in the parent PR, I hope it's ok now